### DISCUSSION TITLE
How to train lemmatizer? Are lookup tables required?

### DISCUSSION THREAD

**Discussion by vistamou** [explosion/spaCy#9711] [Language Support]
Hello, I want to train a lemmatizer for a new language and according to 
the documentation provided on the site this is possible by providing ' lookup tables'--
is this necessary step or there is another way to achieve that?

**Comment by polm** [comment]
Currently there is no trainable lemmatizer in spaCy - there's just lookup based lemmatizers and language specific rule-based lemmatizers. So if you can't provide lookup tables you could implement a lemmatizer yourself, or re-use an existing library for the purpose.

I assume this is related to #9715?

**Reply by polm** [reply]
Note that we have actually just released a trainable lemmatizer, you can read more about it [here](https://explosion.ai/blog/edit-tree-lemmatizer).
